src/window.cpp

```diff
@@ -0,0 +1,73 @@
+#include "window.h"
+#include "SFML/Window.hpp"
+
+Window::Window(int screenWidth, int screenHeight, int frameRate) 
+{
+    window.create(sf::VideoMode(screenWidth, screenHeight), "Flappy Bird");
+
+    sf::Vector2u windowDimensions = window.getSize();
+    pipes = new Pipes(windowDimensions.x, window.getSize().y, frameRate);
+
+    sf::Vector2f center = window.getView().getCenter();
+    bird = new Bird(center, frameRate);
+
+    this->frameRate = frameRate;
+    this->frameTime = sf::seconds(1.0f / frameRate);
```

Comment:
It's not typical in C++ to use `this->member` in constructors the same way as it is in java. Typically some good coding standards with either require you to prefix member variables with `m_xxx` or they will prefix the function parameters with `in`. For example, I tend to use `InFunctionParam` to distinguish beween class members and constructor inputs when they have the same name:

```
Window::Window(int InScreenWidth, int InScreenHeight, int InFrameRate) 
{
    // ...
    frameRate = InFrameRate;
    frameTime = sf::seconds(1.f / frameRate);
}
```